libnavigation-ui/src/test/java/com/mapbox/navigation/ui/map/NavigationSymbolManagerTest.java

```diff
@@ -42,7 +56,7 @@ public void removeAllMarkerSymbols_previouslyAddedMarkersRemoved() {
   }
 
   @Test
-  public void addDestinationMarkerFor_previouslyAddedMarkersRemoved() {
+  public void addCustomSymbolFor_symbolManagerCreatesSymbol() {
```

Comment:
similar to my previous comment, can we consider add a new test case, which verifies:
1. add custom symbol by `addCustomSymbolFor`
2. then `addDestinationMarkerFor`

to verify if the delete old symbol gets executed?